crates/db_schema/src/lib.rs

```diff
@@ -18,6 +22,45 @@ use url::Url;
 pub mod schema;
 pub mod source;
 
+#[derive(Debug, Copy, Clone, Hash, Eq, PartialEq, Serialize, Deserialize, DieselNewType)]
+pub struct PostId(pub i32);
+
+impl fmt::Display for PostId {
+  fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+    write!(f, "{}", self.0)
+  }
+}
+
+#[derive(Debug, Copy, Clone, Hash, Eq, PartialEq, Serialize, Deserialize, DieselNewType)]
+pub struct PersonId(pub i32);
```

Comment:
This is the only really important change worth looking at here, cause everything else is boilerplate. These fields add `DieselNewType`, which implements a lot of the `FromSql` / `ToSql` things for you.

Reply:
I hope this doesnt increase compilation time too much. Will benchmark it again once we are close to releasing.